src/Npgsql/NpgsqlTypes/NpgsqlDbType.cs

```diff
@@ -24,49 +24,42 @@ public enum NpgsqlDbType
     /// Corresponds to the PostgreSQL 8-byte "bigint" type.
     /// </summary>
     /// <remarks>See https://www.postgresql.org/docs/current/static/datatype-numeric.html</remarks>
-    [BuiltInPostgresType("int8", PostgresTypeOIDs.Int8)]
```

Comment:
Should be able to remove BuiltInPostgresTypeAttribute no?

Reply:
Yeah, true.